web_src/js/features/formatting.js

```diff
@@ -0,0 +1,15 @@
+import {prettyNumber} from '../utils.js';
+
+const {lang} = document.documentElement;
```

Comment:
IIRC, `document.documentElement.lang` comes from backend rendered `<html lang=xxx>`, but we only have limited i18n names in backend. Should we use browser default i18n settings?

Reply:
Would only use browser's language in case `lang` is absent. Users need to be able to switch to any language.